python/viewer/image_viewer.py

```diff
@@ -107,7 +109,7 @@ def main():
                     break
 
                 # create rgb image from point cloud / SAR
-                im_rgb = to_rgb_image(render_data)
+                im_rgb = to_rgb_image(render_data, args.color_by_elevation)
```

Comment:
I think it makes more sense for me to have to_rgb_image takes in a color map instead of a boolean. You can choose which colormap to apply in this line.

Reply:
Do you have a suggestion on how if we input a colormap to `to_rgb_image` we would choose later which value to use in the color map ? I can look at `cmap_elevation.cmap.name`, but I don't really like that since it is not super clear... Should I create a special class from `ScalarMappable` with a proper name field ?